packages/app/src/navigation/SidebarNavigation.vue

```diff
@@ -28,7 +28,6 @@
     </button>
     <div class="flex flex-col flex-1 ">
       <SidebarNavigationHeader
-        v-if="props.gql"
```

Comment:
Always render the component even if the query isn't finished yet - handle the loading state inside of the component